src/main/resources/db/migration/data/dev/R__2_3__moves.sql

```diff
@@ -1,10 +1,10 @@
-insert into moves (move_id, cancellation_reason, cancellation_reason_comment, drop_off_or_cancelled, from_nomis_agency_id, move_date, move_type, notes, pick_up, profile_id, reference, report_from_location_type, report_to_location_type, status, supplier, to_nomis_agency_id, updated_at, vehicle_registration) select * from (select 'M20', null, null, to_timestamp('2020-12-02 14:20:30.000000', 'YYYY-MM-DD HH24-MI-SS.US'), 'WYI', date('2021-02-28'), 'REDIRECTION', 'MoveRedirect: Redirect event MoJ''s fault', to_timestamp('2020-12-02 10:20:30.000000', 'YYYY-MM-DD HH24-MI-SS.US'), 'PR3', 'REDIRMOVE', 'prison', 'prison', 'completed', 'GEOAMEY', 'GNI', to_timestamp('2020-06-16 10:20:30.000000', 'YYYY-MM-DD HH24-MI-SS.US'), '123, 123') as tmp where not exists (select move_id from moves where move_id = 'M20');
-insert into moves (move_id, cancellation_reason, cancellation_reason_comment, drop_off_or_cancelled, from_nomis_agency_id, move_date, move_type, notes, pick_up, profile_id, reference, report_from_location_type, report_to_location_type, status, supplier, to_nomis_agency_id, updated_at, vehicle_registration) select * from (select 'M30', null, null, to_timestamp('2020-12-04 11:20:30.000000', 'YYYY-MM-DD HH24-MI-SS.US'), 'WYI', date('2021-02-28'), 'LONG_HAUL', '', to_timestamp('2020-12-03 12:00:30.000000', 'YYYY-MM-DD HH24-MI-SS.US'), 'PR1', 'LONGMOVE', 'prison', 'prison', 'completed', 'GEOAMEY', 'BOG', to_timestamp('2020-06-16 10:20:30.000000', 'YYYY-MM-DD HH24-MI-SS.US'), '123, 456') as tmp where not exists (select move_id from moves where move_id = 'M30');
+insert into moves (move_id, cancellation_reason, cancellation_reason_comment, drop_off_or_cancelled, from_nomis_agency_id, move_date, move_type, notes, pick_up, profile_id, reference, report_from_location_type, report_to_location_type, status, supplier, to_nomis_agency_id, updated_at, vehicle_registration) select * from (select 'M20', null, null, to_timestamp('2020-12-02 14:20:30.000000', 'YYYY-MM-DD HH24-MI-SS.US'), 'PRISON1', date('2021-02-28'), 'REDIRECTION', 'MoveRedirect: Redirect event MoJ''s fault', to_timestamp('2020-12-02 10:20:30.000000', 'YYYY-MM-DD HH24-MI-SS.US'), 'PR3', 'REDIRMOVE', 'prison', 'prison', 'completed', 'GEOAMEY', 'GNI', to_timestamp('2020-06-16 10:20:30.000000', 'YYYY-MM-DD HH24-MI-SS.US'), '123, 123') as tmp where not exists (select move_id from moves where move_id = 'M20');
```

Comment:
I have tweaked the test data as we cannot guarantee the order in which the tests are executed.  There was a clash with a test that updates locations.